plugins/flytekit-flyin/Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG PYTHON_VERSION
+FROM python:${PYTHON_VERSION}-slim-buster
+MAINTAINER Flyte Team <users@flyte.org>
+LABEL org.opencontainers.image.source https://github.com/flyteorg/flytekit
+WORKDIR /root
+ENV PYTHONPATH /root
+
+# Install necessary packages
+RUN apt-get update \
```

Comment:
maybe add some comments about what these commands do. it looks a bit overwhelming now

Reply:
no problem!